app/decorators/coercion_decorator.rb

```diff
@@ -0,0 +1,39 @@
+class CoercionDecorator
```

Comment:
I might be better able to articulate this in a pairing session but this in my opinion needs a little thought. My earlier point was not conveyed clearly which is my fault and I think would made clearer if we think duck typing and documentation and by bolstering the unit spec.